data/repos.yml

```diff
@@ -179,6 +225,13 @@
   sentry_url: false
   dashboard_url: false
 
+  # Looks active
+- repo_name: datagovuk-visual-regression-tests
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
```

Comment:
This is active too I think @kentsanggds ?

Reply:
Yes, used for testing visual differences between CKAN versions but also useful for detecting problems with the user flow